prql-compiler/tests/integration-vendors/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3"
+
+services:
+  postgres:
+    image: "postgres:15-alpine"
```

Comment:
Is there any reason to specify version 15?

```suggestion
    image: "postgres:alpine"
```

Reply:
No specific reason. I just took the latest version

Reply:
Does dependabot upgrade these? If it does, that's ideal — then we lock versions but they don't fall out of date.

(BTW fine to refine these sorts of things in another PR)

Reply:
I don't know if dependabot checks these files but the latest version should be fine